athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/serde/v2/ArrowTypeSerDe.java

```diff
@@ -684,4 +686,40 @@ protected ArrowType doTypedDeserialize(JsonParser jparser, DeserializationContex
             }
         }
     }
+
+    private static final class MapSerDe
+    {
+        private static final String KEY = "key";
+        private static final String VALUE = "value";
+
+        private static final class Serializer extends TypedSerializer<ArrowType>
+        {
+            private Serializer()
+            {
+                super(ArrowType.class, ArrowType.Map.class);
+            }
+
+            @Override
+            protected void doTypedSerialize(ArrowType arrowType, JsonGenerator jgen, SerializerProvider provider)
+                    throws IOException
+            {
+                // no fields
+            }
+        }
+
+        private static final class Deserializer extends TypedDeserializer<ArrowType>
+        {
+            private Deserializer()
+            {
+                super(ArrowType.class, ArrowType.Map.class);
+            }
+
+            @Override
+            protected ArrowType doTypedDeserialize(JsonParser jparser, DeserializationContext ctxt)
+                    throws IOException
+            {
+                return new ArrowType.Map(false);
```

Comment:
What does passing `false` to the constructor do?  Might be worth adding this as a constant with more context.

Reply:
false just means that the entries are not ordered. I can add a comment.